src/main/java/bio/terra/tanagra/query/filtervariable/BooleanAndOrFilterVariable.java

```diff
@@ -17,23 +17,22 @@ public BooleanAndOrFilterVariable(LogicalOperator operator, List<FilterVariable>
   }
 
   @Override
-  protected String getSubstitutionTemplate() {
+  protected String getSubstitutionTemplate(CloudPlatform platform) {
+    // FIXME: throw ??
```

Comment:
Just checking in on this FIXME? Do we want to change this to TODO or fix it?

Reply:
I can remove this FIXME. I noticed the `return null` when I was looking at the code and it seems suspicious to me, but that's what the original code is doing. https://github.com/DataBiosphere/tanagra/blob/main/underlay/src/main/java/bio/terra/tanagra/query/filtervariable/BooleanAndOrFilterVariable.java#L21